src/WishlistView/WishlistView.scss

```diff
@@ -0,0 +1,35 @@
+.WishlistView {
+
+  .WishlistViewItem {
+    margin: 20px 0;
+  }
+
+  .ImageBlock {
+    position: relative;
+    width: 200px;
+  }
+  .ImageBlock--Overlay {
+    position: absolute;
+    top: 25%;
+    left: 0;
+    width: 100%;
+  }
+
+  .ImageBlock--OverlayText {
+    color: white;
+    font-size: 20px;
+    background: rgb(0, 0, 0); /* fallback color */
+    background: rgba(0, 0, 0, 0.7);
```

Comment:
`rgb` and `rgba` ....one of this is for fallback, I'm not completely familiar with sass but I think this should be handled gracefully using some syntactic sugar like a mixin or maybe even by default?